content/references/protocol-reference/transactions/transaction-types/clawback.md

```diff
@@ -0,0 +1,64 @@
+---
+html: clawback.html
+parent: transaction-types.html
+blurb: Claw back an issued currency transfer between accounts in a trust set.
+labels:
+  - Tokens
+status: not_enabled
+---
+# Clawback
+
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/ripple/app/tx/impl/Clawback.cpp "Source")
+
+<embed src="/snippets/_clawback-disclaimer.md" />
+
+Claw back an issued currency transfer between accounts in a trust set.
```

Comment:
Lot of problems with this sentence:

- There is no concept of "a trust set". The "set" in "TrustSet" is a verb, not a noun (the source file is even `SetTrust.cpp`!)
- You're clawing back a balance, not a transfer. The tokens may have been transferred among users several times between issuing and clawback.
- We prefer to call them "tokens" not "issued currency"
- Ideally, the first sentence of an article should have an "escape hatch" link that goes back to the concept page explaining the greater context for the feature.

```suggestion
[Claw back tokens](clawing-back-tokens.html) issued by your account. This only works if you have enabled the Allow Clawback flag before issuing the tokens.
```

Reply:
I don't agree that there should be an escape hatch link. Thank you, Rome, for your thorough review.